gpkit/small_scripts.py

```diff
@@ -56,6 +56,8 @@ def unitstr(units, into="%s", options="~", dimless='-'):
     if units and not isinstance(units, Strings):
         try:
             rawstr = ("{:%s}" % options).format(units)
+            if str(units.units) == "count":
+                rawstr = "1.0 count"
```

Comment:
I don't understand why this special case is needed. Using the version on master (without the change above), unitstr seems to work fine:

```python
In [1]: from gpkit import units

In [2]: from gpkit.small_scripts import unitstr

In [3]: from gpkit import Variable

In [4]: x = Variable("x", "ft")

In [5]: x.units
Out[5]: <Quantity(1, 'foot')>

In [6]: unitstr(x)
Out[6]: 'ft'

In [7]: x = Variable("x", "count")

In [8]: unitstr(x)
Out[8]: 'count'
```